library/Makefile

```diff
@@ -91,9 +104,7 @@ shared: libmbedcrypto.$(DLEXT) libmbedx509.$(DLEXT) libmbedtls.$(DLEXT)
 # tls
 libmbedtls.a: $(OBJS_TLS)
 	echo "  AR    $@"
-	$(AR) -rc $@ $(OBJS_TLS)
-	echo "  RL    $@"
-	$(AR) -s $@
+	$(AR) $@ $(OBJS_TLS)
```

Comment:
I believe the equivalent to 'ar -s ' in IAR is `iarchive --symbols`
Please try it, and if it works, have this AR flags according to the toolchian:
e.g.
~~~
on the top:
+ifeq ($(CC),iccarm)
ARS = --symbols
else
ARS = -s
endif
in the command:
$(AR) $(ARS) $@
~~~
or something similar. If it's not the equivalent, find a different way to run this indexing command, on non IAR toolchain